meta-mender-demo/recipes-extended/images/mender-demo-minimal-rw.bb

```diff
@@ -0,0 +1,4 @@
+inherit core-image
+
+IMAGE_FEATURES += "ssh-server-openssh allow-empty-password debug-tweaks"
+IMAGE_INSTALL_append = " example-custom-inventory"
```

Comment:
I support the idea of having such an image, but I'm a bit worried about discoverability and maintenance. Should we use this as the build target in our docs?

Also, since you're adding `example-custom-inventory`, you might consider adding `example-state-scripts` as well.

Reply:
> Should we use this as the build target in our docs?

Not yet, this is just a simplifies and aims to standardize generation of demo images.
I wasn't aiming for it to be used beyond that for now, but do hope to see it go in this direction eventually.

Reply:
I've also removed the `example-custom-inventory` from the image as that will never make sense beyond anything but giving demos.